index.html

```diff
@@ -733,195 +747,213 @@
 
-Slide properties serve multiple purposes:
+Sposób komunikacji z serwerem:
 
-* Naming and styling slides using properties `name` and `class`
+- wykorzystanie kodu generowanego na podstawie definicji API
 
-* Using slides as templates using properties `template` and `layout`
+- asynchroniczne zapytania do API w ramach zakresu widoków
 
-* Expansion of `{{ property }}` expressions to property values
-
-See the [complete 
-list](https://github.com/gnab/remark/wiki/Markdown#slide-properties) of 
-slide properties.
+- lekkie wykonywanie działań w tle
 ]
 ---
 .left-column[
-  ## Markdown extensions
-  ### - Slide properties
-  ### - Content classes
+  ## Powitanie
 ]
 .right-column[
-Any occurences of one or more dotted CSS class names followed by square 
-brackets are replaced with the contents of the brackets with the 
-specified classes applied:
 
-```remark
-.footnote[.red.bold[*] Important footnote]
-```
+.text-content[
 
-Resulting HTML extract:
+Dwie podstawowe ścieżki wykorzystania aplikacji:
 
-```xml
-&lt;span class="footnote"&gt;
-  &lt;span class="red bold"&gt;*&lt;/span&gt; Important footnote
-&lt;/span&gt;
-```
+- wyszukiwanie miejsca na mapie
+
+- wyszukiwanie tesktowe miejsca
+
+]
+
+.image-content[
+<img src="./img/hello_light.jpg" alt="hello" height="400px"/>
+]
 ]
 ---
 .left-column[
-  ## Markdown extensions
-  ### - Slide properties
-  ### - Content classes
-  ### - Syntax Highlighting
+  ## Powitanie
+  ## Mapa
 ]
 .right-column[
-Code blocks can be syntax highlighted by specifying a language from the 
-set of [supported 
-languages](https://github.com/gnab/remark/wiki/Configuration#highlighting).
 
-Using
- [GFM](http://github.github.com/github-flavored-markdown/) fenced code 
-blocks you can easily specify highlighting language:
+.text-content[
 
-.pull-left[
+Możwliwość wyszukiwania na mapie:
 
-&lt;pre&gt;&lt;code&gt;```javascript
-function add(a, b)
-  return a + b
-end
-```&lt;/code&gt;&lt;/pre&gt;
-]
-.pull-right[
+- podstwowe informacje o miejscach
 
-&lt;pre&gt;&lt;code&gt;```ruby
-def add(a, b)
-  a + b
-end
-```&lt;/code&gt;&lt;/pre&gt;
-]
+- dostęp do lokalizacji obiektów
 
-A number of highlighting 
-[styles](https://github.com/gnab/remark/wiki/Configuration#highlighting)
- are available, including several well-known themes from different 
-editors and IDEs.
+- możliwość skorzystania z GPS urządzenie
+
+]
 
+.image-content[
+<img src="./img/map_light.jpg" alt="map" height="400px"/>
+]
 ]
 ---
 .left-column[
-  ## Presenter mode
+  ## Powitanie
+  ## Mapa
+  ## Widok podstawowy
 ]
 .right-column[
-To help out with giving presentations, a presenter mode comprising the
-following features is provided:
 
-- Display of slide notes for the current slide, to help you remember
-  key points
+.text-content[
 
-- Display of upcoming slide, to let you know what's coming
+Prezentacja danych dostarczonych z serwera:
+
+- czytelna i prosta prezentacja podstawowych danych
+
+- reaktywne ładowanie poszczególnych elemntów
```

Comment:
reaktywne? xD
responsywne?

Reply:
chodzilo o reaktywne, ale moze to zbyt specjalistyczne slowo, w sensie nei powrzechnie uzywane, napisze jakies niezalezne czy cos